### DISCUSSION TITLE
Tankstellen nur an der Route Anzeigen

### DISCUSSION THREAD

**Discussion by 9Biker** [osmandapp/OsmAnd#18802] [Q&A]
Tankstellen nur an der Route Anzeigen
Kann/könnte manTankstellen nur in unmittelbarer nähe an einer GPX Route sich anzeigen lassen.

Show gas stations only on the route
Is it possible to display petrol stations only in the immediate vicinity of a GPX route?

**Comment by mikehgentry** [comment]
There was some related discussion in #18007
